awaitility-groovy/pom.xml

```diff
@@ -54,6 +54,15 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
I only added this to be sure my extension method works. Before I did the last commit -- changing the extension method from until(Closure) to until(Runnable) --, mvn install did work, but running the two new tests alone (e.g. by using mvn install -Dtest=...) did not. 

This was, because by pure luck the other tests where running first and did call AwaitilityGroovyBridge.initializeAwaitilityGroovySupport().

To ensure the tests work the same regardless of build order, I decided to run every test in it's own VM.

End Users are not required to add such a setting to their projects.